Scripts/script-CommonServerPython.yml

```diff
@@ -402,31 +671,78 @@ script: |-
 
 
   def fileResult(filename, data):
+      """
+         Creates a file from the given data
+
+         :type filename: ``str``
+         :param filename: The name of the file to be created (required)
+
+         :type data: ``str``
+         :param data: The file data (required)
+
+         :return: A Demisto war room entry
+         :rtype: ``dict``
+      """
       temp = demisto.uniqueFile()
       with open(demisto.investigation()['id'] + '_' + temp,'wb') as f:
           f.write(data)
       return {'Contents': '', 'ContentsFormat': formats['text'], 'Type': entryTypes['file'], 'File': filename, 'FileID': temp}
 
 
   def file_result_existing_file(filename, saveFilename = None):
-      """ Create a file result from an existing file by moving it into the right file name """
+      """
+         Rename an existing file
+
+         :type filename: ``str``
+         :param filename: The name of the file to be modified (required)
+
+         :return: A Demisto war room entry
+         :rtype: ``dict``
+      """
       temp = demisto.uniqueFile()
       os.rename(filename, demisto.investigation()['id'] + '_' + temp)
       return {'Contents': '', 'ContentsFormat': formats['text'], 'Type': entryTypes['file'],
           'File': saveFilename if saveFilename else filename, 'FileID': temp}
 
 
   def flattenRow(rowDict):
+      """
+         Flatten each element in the given rowDict
+
+         :type rowDict: ``dict``
+         :param rowDict: The dict to be flattened (required)
+
+         :return: A flattened dict
+         :rtype: ``dict``
+      """
       return {k: formatCell(rowDict[k]) for k in rowDict}
 
 
   def flattenTable(tableDict):
+      """
+         Flatten each row in the given tableDict
+
+         :type tableDict: ``dict``
+         :param tableDict: The table to be flattened (required)
+
+         :return: A flattened table
+         :rtype: ``dict``
+      """
       return [flattenRow(row) for row in tableDict]
 
   MARKDOWN_CHARS = "\`*_{}[]()#+-!"
 
 
   def stringEscapeMD(st):
+      """
+         Escape any chars that might break a markdown string
```

Comment:
Very technical, can be it be said: "change the characters that may break a markdown string"?